bin/node-template/node/src/service.rs

```diff
@@ -24,100 +20,97 @@ native_executor_instance!(
 	node_template_runtime::native_version,
 );
 
-/// Starts a `ServiceBuilder` for a full service.
-///
-/// Use this macro if you don't actually need the full service, but just the builder in order to
-/// be able to perform chain operations.
-macro_rules! new_full_start {
+macro_rules! new_full_up_to_import_queue {
```

Comment:
First of all: I think this doesn't need to be a macro anymore. The pattern with the macros here was really dreaded, it will be nice to see that fixed.

Reply:
What do you recommend I replace it with? The best option might be to have a function that returns `ServiceParams`, but I'd need to change that to hold a `task_manager ` and not a `&mut task_manager`, and then pass it through to the `ServiceComponents`.

Reply:
Yes! I think that ServiceParams is exactly what I have in mind later with BaseServiceComponents. Removing the `&mut` makes sense

Reply:
Yes please remove these macros!